jetty-ee9/jetty-ee9-nested/src/test/java/org/eclipse/jetty/ee9/nested/SessionHandlerTest.java

```diff
@@ -73,7 +73,7 @@ public void beforeEach() throws Exception
         _sessionHandler = new SessionHandler();
         _sessionHandler.setSessionCookie("JSESSIONID");
         _sessionHandler.setUsingCookies(true);
-        _sessionHandler.setUsingURLs(false);
+        _sessionHandler.setUsingUriParameters(false);
```

Comment:
Interesting that the ee9 and ee10 SessionHandlerTests have diverged such that this code isn't present in the ee10 version.

Reply:
I think it just assumed it was true.  Added explicit set to the appropriate test.